tests/test_parseutils.py

```diff
@@ -1,14 +1,14 @@
 import pytest
-from pgcli.packages.parseutils import extract_tables
-from pgcli.packages.parseutils import find_prev_keyword, is_open_quote
+from pgcli.packages.parseutils import (extract_tables, TableReference,
+    find_prev_keyword, is_open_quote)
 
 def test_empty_string():
     tables = extract_tables('')
     assert tables == ()
 
 def test_simple_select_single_table():
     tables = extract_tables('select * from abc')
-    assert tables == ((None, 'abc', None, False),)
+    assert tables == (TableReference(None, 'abc', None, False),)
```

Comment:
That was just me being confused by the output of the tests, so I've reverted it now.